tmt/steps/__init__.py

```diff
@@ -68,8 +68,25 @@
 
 
 DEFAULT_ALLOWED_HOW_PATTERN: Pattern[str] = re.compile(r'.*')
-DEFAULT_PLUGIN_METHOD_ORDER: int = 50
 
+#
+# Following are default and predefined order values for various special phases
+# recognized by tmt. When adding new special phase, add its order below, and
+# do not forget to update the list in /spec/core/order. Please, keep the name
```

Comment:
This `/spec/core/order` part is outdated and should be dropped from the comment.

Reply:
Nope, fixed in 8e3586a6